test/functionalTest/cases/2670_milliseconds_for_DateTime_attrs/2670_milliseconds_for_DateTime_attrs.test

```diff
@@ -0,0 +1,90 @@
+# Copyright 2020 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Milliseconds for a DateTime attribute
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB 0
+
+--SHELL--
+
+#
+# 01. Create an Entity E1, with an attribute 'ts' that is of DateTime type and with a value with milliseconds
+# 02. GET E1 and see its 'ts' attribute with its milliseconds
+#
+
+echo "01. Create an Entity E1, with an attribute 'ts' that is of DateTime type and with a value with milliseconds"
+echo "==========================================================================================================="
+payload='{
+  "id": "E1",
+  "ts": {
+    "type": "DateTime",
+    "value": "2020-07-03T09:24:04.456"
```

Comment:
Although tis is a format perfectly valid in ISO8601, for completness, I'd suggest to add another cases using:

```
"2020-07-03T09:24:04.456Z" (to test Z timezone)
"2020-07-03T09:24:04.456+0100" (to test a not Z timezone)
```

Reply:
This is a little bit out of scope for this PR.
I'll test it - if it works, then all good, I'll include it. If it does not work, then I'd propose to create an issue for it.
Has this ever been tested in Orion?

Reply:
Yes. Orion supports timezones and we have .test covering it (as far as I remember). See https://fiware-orion.readthedocs.io/en/master/user/ngsiv2_implementation_notes/index.html#datetime-support

Reply:
Yes, you are right - it works.
Fixed in e8a770d